swap_meet/vendor.py

```diff
@@ -1,2 +1,99 @@
+import pytest
```

Comment:
Looks like this was accidentally imported. It's good practice when you're done coding to make sure VS Code didn't automatically include imports you weren't expecting.